File: lab03/lab03task02/src/lab03_task02_node.cpp
#include "ros/ros.h"
//TODO: Include the header files for three services defined in lab03task02srv

//TODO: Complete these three functions
bool convert_quat2zyx();
bool convert_quat2angleaxis();
bool convert_rotmat2quat();

int main(int argc, char **argv)
{
    ros::init(argc, argv, "rotation_converter");
    ros::NodeHandle nh;

    //TODO: Define three services

    ros::spin();
    return 5;
}

File: lab03/lab03task02/scripts/lab03_task02_node.py
#!/usr/bin/env python

import rospy
import numpy as np
##TODO: Include the header file for the three services defined in lab03task02srv

##TODO: Complete these functions
def convert_quat2zyx():
    print 'test'


def convert_quat2angleaxis():
    print 'test'


def convert_rotmat2quat():
    print 'test'


def rotation_converter():
    rospy.init_node('rotation_converter')
    ##TODO: Initialise the services
    rospy.spin()


if __name__ == "__main__":
    rotation_converter()
